tidy3d/web/api/webapi.py

```diff
@@ -134,6 +139,18 @@ def run(
     :meth:`tidy3d.web.api.container.Batch.monitor`
         Monitor progress of each of the running tasks.
     """
+    if use_local_cache and Path(path).is_file():
+        local_sim = SimulationData.from_file(path)
```

Comment:
Maybe better to put this whole thing in a try-except and continue with the run if something goes wrong.